### PR TITLE
wdio-runner: at the end of process in session clean-up - mute any request-related RuntimeError()

### PR DESCRIPTION
**Pull request by Eduard-iCH**
## Proposed changes

Currently if selenium session is already terminated for some reason or selenium-server is dead - **runner.js** crash at **this.endSession()** with **RuntimeError("No active session with ID ..." / "ECONNREFUSED" / etc)**. This lead to **event('runner:end')** _**never**_ get fired and break reporting.

Now it's just try to mute as much http-request errors as it can.

Before / After: https://imgur.com/a/s5QtXXN

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
Little code changes inside session->"DELETE" command. It's should have **zero** impact on other code.
I checked "test:functional" and test:wdio" - they seems already little broken, but no new errors appeared. It's not perfect solution through, but better than nothing.

Still looks like there is a problem (?) - wdio does not always kill opened browser (chromedriver / chrome) instances. But probably it's selenium problem.

[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by Eduard-iCH** [comment]
I start to think maybe it's better to mute any http-request error here?

**Comment by Eduard-iCH** [comment]
Reworked this fix to mute as much errors as it can. Description updated.

**Comment by Eduard-iCH** [comment]
Trailing spaces removed.

File: lib/protocol/session.js [hunk 1]
```diff
@@ -87,6 +87,20 @@ export default function session (doWhat = 'GET', sessionId) {
...
+                /**
+                 * At the end of runner - mute any request-related error here (session not exist, connection refused, etc ...)
+                 * Done in order to not break the runner, when selenium-part is broken for some reason.
+                 */
+                this.requestHandler.sessionID = null
+                return {
+                    value: [],
+                    _status: 500
+                }
+            } else {
```
Comment (christian-bromann) [inline lib/protocol/session.js]: no else needed

Comment (Eduard-iCH) [reply lib/protocol/session.js]: Done

**Comment by Eduard-iCH** [comment]
Done

**Pull request closed without merging**